project_info.Rmd

```diff
@@ -0,0 +1,82 @@
+ecological data and rigorous scientific methods. Groups are expected to hypothesize
+about their chosen data, examine their hypotheses with reproducible and quantitative
+analysis techniques, visualize their results, and create scientific products in the
+form of a report and a presentation.
+
+[This paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210691/) was written by a
+group of students as part of the first version of this course, created by Dr. 
+Christie Bahlai. 
+
+### Data
+
+A list of recommended datasets can be found [here](). You are welcome to choose a
+dataset not listed, or data collected as part of a research project, but keep in mind
+that you may not submit anything twice: any work you do as part of this course may not
+be submitted for credit in another course (such as a fourth-year research project) and vice versa. 
+If choosing a dataset not listed, make sure it is well-documented, legitimate, and 
+complex enough to support your analysis efforts. Your work should be original; your
+project should not be a reproduction of published analysis. 
+
+### Project deliverables
+
+The following components will be graded as part of the project:
+
+1. Weekly project updates:
+    * These will take place on Tuesdays at the beginning of class
+    * Two people will present from the group, rotating each week 
+   (each member will present 3-4 times over the course of the project)
+    * Each group update should take no more than 2 minutes
+    * These can be very informal: no slides or visuals are necessary 
+   (but you can include them if you want to)
+    * You will not be marked on correctness; you are encouraged to be open about 
+   any difficulties your group is facing so that other groups and the instructors
+   can provide helpful feedback
+  
+2. Report styled as a journal article, with these or similar sections: 
+    * Introduction 
+    * Methods
+    * Results 
+    * Discussion 
+    * Code: project results must be reproducible by someone else
+   
+3. 10 minute presentation with 2 minutes for questions, styled as a conference
+presentation (assume not too much familiarity with the topic in the audience).
+The presentations will be held on the last day of class (Dec 4).
+
+4. Self-assessed contributions: maximum 1 page, outlining individual contributions
+(code, writing, presenting, etc). Each group member will submit their own
+self-assessment.
+
+While you may not submit your work for this course for credit in another course, you
+are welcome to publish or present your work in an academic setting. Groups are
+encouraged to publish their work on [figshare](https://figshare.com/), an open, 
+citable repository of scientific content. 
+
+All items (except the presentation) are due on December 5th at 11:59 pm.
+
+## Project submission
+
+The project report and code should be submitted on GitHub - each group will have 
+their own GitHub repository to which you can upload your report and code. 
+
```

Comment:
Should we mention something about the format for the final report? For example, should they have Rmd and pdf files on their GitHub or will we accept word documents?

Reply:
That's something I want to discuss at our meeting - we want it to be reproducible, so should we make them write the whole report as an R markdown document? Then they would submit pdf or html plus .Rmd.